layers/+lang/python/packages.el

```diff
@@ -95,6 +96,19 @@
       (add-to-list 'spacemacs-jump-handlers-python-mode
                    '(anaconda-mode-find-definitions :async t)))))
 
+(defun python/init-code-cells ()
+  (use-package code-cells
+    :if (not (configuration-layer/layer-used-p 'ipython-notebook))
+    :defer t
+    :init
+    (progn
+      (add-hook 'python-mode-hook 'code-cells-mode)
+      (spacemacs/set-leader-keys-for-major-mode 'python-mode
+        "gp" 'code-cells-backward-cell
+        "gn" 'code-cells-forward-cell
+        "sc" 'code-cells-eval
+        "sa" 'code-cells-eval-above))))
```

Comment:
These bindings should be listed in the README. Also you should check that there are no conflicts with the standard bindings from LSP layer if you bind them unconditionally.

Reply:
Hmmmmm looks like this Readme is out of date and there are mismatches between xref and peek bindings. The layer must be a revised.

Back to your PR I have quickly checked the bindings and "gn" is already taken when peek bindings replace xref bindings.

"qb" is taken by xref bindings.

I would suggest using "qN" and "qB" these are free for both jump solutions.

Reply:
Sounds cool, lets go with your bindings 👍